pkgs/by-name/so/soarca/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  go-swag,
+  nixosTests,
+  nix-update-script,
+}:
+
+buildGoModule {
+  pname = "soarca";
+  version = "1.1.0-beta-1-unstable-2024-12-19";
```

Comment:
This version is based on a GitHub tag/release, so no need of unstable.

Also, you can use this version in rev.

Reply:
I tried to package the latest available version (with some bugfixes), which strictly speaking isn't the tag `1.1.0-beta-1`, but a newer, untagged version. I followed [the versioning guide](https://github.com/NixOS/nixpkgs/blob/38bbe73f0d0a9af38bd21a915398689560ef0a9f/pkgs/README.md#versioning) as closely as possible, but let me know and I'll change it. This is my second ever contribution to Nixpkgs :)

Reply:
Got it